ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,114 @@
+# ADR-00002 Personas related to Managed Services
+
+Authors: Craig Robinson
+
+
+## Status
+
+Draft
+
+
+## Problem Statement
+
+A successful SRE practice requires a cross-functional group of stakeholders to engage and work across teams to focus on reliability. This document lists the Personas involved in supporting a cloud service. For example, as a stakeholder of SLOs, I want to be able to consume the data in a relevant and consistent way in accordance with my "persona". The roles and responsibilities of each Persona will be documented in RACI format in a future PR.
+
+## Goals
+
+This ADR hopes to resolve the problem statement by identifying and documenting the key personas in a summary format. The output will be a document which will be referenced from other assets.
+
+
+## Non-goals
+
+* This ADR does not aim at capturing every form or type of persona. It is purposefully generalised, utilising the most frequent persona types as experienced within the Red Hat 'Managed Services' context. The reason is that every organisation calls their personas by different names (and they will inevitably have different role descriptions). Nevertheless, this document hopes to capture the main types in a translatable way.
+
+
+## Current Architecture
+
+* There was some pre-existing art in an internal Red Hat document entitled the "Application Services Primer". That was used as a source for this ADR.
+
+
+## Proposed Architecture
+
+The proposed content is as follows:
+
+***
+
+**Personas related to "Managed Services"**
+
+**Purpose**
+To underpin many other Site Reliability Engineering-related topics, we must define the group of "personas" describing  the interaction of that person (or group of people) with an environment supported by SREs.
+
+**Service Owner**
```

Comment:
@brizrobbo Still need to see headers for these personas. This one would be `### Service Owner` with the others following the same h1, h2, h3, h4 etc heading/sub-heading. This is important because we will want to link directly to this persona, and that will be done for us automatically with this recommendation.

Reply:
@lisa I may be confused with how the process is supposed to work - but this is just the ADR. Isn't the idea that, once the ADR is approved, then we would just create another document that contains the "approved content" from this ADR?

Something like https://github.com/operate-first/sre/personas.md (note the path and filename)?

Have I got it wrong?

Reply:
Based on a comment from @jeremyeder in another channel, I realize that this ADR is representative of the actual content. Therefore, I have refactored the headings to bring them inline with the rest of the document.